general/g.version/main.c

```diff
@@ -140,26 +144,18 @@ int main(int argc, char *argv[])
     }
 
     if (gish_rev->answer) {
-	char **rev_ver = G_tokenize(GIS_H_VERSION, "$");
-	char **rev_time = G_tokenize(GIS_H_DATE, "$");
-	const int tokens_expected = 3;
+	char *rev_ver = GIS_H_VERSION;
+	char *rev_time = GIS_H_DATE;
 	int no_libgis = FALSE;
 
-	/* if number of tokes is right, print it */
-	if (G_number_of_tokens(rev_ver) == tokens_expected &&
-	    G_number_of_tokens(rev_time) == tokens_expected) {
+	if (*rev_ver && *rev_time) {
 	    if (shell->answer) {
-                const char *p;
-                p = strstr(rev_ver[1], " ");
-		fprintf(stdout, "libgis_revision=%s\n",
-			p ? p + 1 : "00000");
-                p = strstr(rev_time[1], " ");
-		fprintf(stdout, "libgis_date=\"%s\"\n",
-			p ? p + 1 : "?");
+		fprintf(stdout, "libgis_revision=\"%s\"\n", rev_ver);
+		fprintf(stdout, "libgis_date=\"%s\"\n", rev_time);
 	    }
 	    else {
-		fprintf(stdout, "libgis %s\nlibgis %s\n", rev_ver[1],
-			rev_time[1]);
+		fprintf(stdout, "libgis %s\nlibgis %s\n", rev_ver,
```

Comment:
@metzm Let's change 

```
libgis 5872054b4
libgis Thu Mar 12 22:06:12 2020 +0100
```

to

```
libgis revision: 5872054b4
libgis date: Thu Mar 12 22:06:12 2020 +0100
````

or similar

Reply:
Agreed, applied in 5872054